oe_theme.theme

```diff
@@ -1614,3 +1614,12 @@ function oe_theme_preprocess_patterns_overview_page(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_paragraph__oe_contact(&$variables) {
+  if (!$variables['paragraph']->get('field_oe_title')->isEmpty()) {
```

Comment:
We can use `$variables['title'] = $variables['paragraph']->get('field_oe_title')->value;`without condition - if it is empty it returns empty.